lambda_function.py

```diff
@@ -0,0 +1,80 @@
+import json
+import urllib.parse
+import boto3
+import logging
+#from botocore.exceptions import ClientErro
+
+print('Loading function') 
+region = 'us-east-1'
+#bucket_name = 'csv-bucket-3534r35d'
+csv_bucket_name = 'bucket-for-converted-csv-objects'
+csv_object_name = 'converted-csv-object'
+
+
+recieved_object_name = ""
+response = ""
+s3 = boto3.client('s3')
+s3_client = boto3.client('s3')
+
+def lambda_handler(event, context):
+    #print("Received event: " + json.dumps(event, indent=2))
+
+    buildCSV = ""
+    # Get the object from the event and show its content type
+    bucket = event['Records'][0]['s3']['bucket']['name']
```

Comment:
while processing this kind of event arrays, always loop through the records instead of indexing a specific one

instead of event['Records'][0] use a for loop and iterate through event['Records']